apps/allocations/app/components/App/AllocationsHistory.js

```diff
@@ -74,7 +74,7 @@ const AllocationsHistory = ({ allocations }) => {
         return recipients.map((recipient, index) => {
           const allocated = BigNumber(recipient.supports).div(totalSupports)
           return (
-            <div key={index}>
+            <RecipientContainer key={index}>
```

Comment:
While we're here, I'd love to get rid of the unnecessary `index` variable and use `key={recipient.candidateAddress}` instead.